### PR TITLE
tema - clase si obiecte

### PR DESCRIPTION
**Pull request by flavia-c**
pull request.

### CODE REVIEW

File: capuccino/capuccino_py.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from math import *
+from espresso.espresso_py import espresso
+
+class capuccino(espresso):
```
Comment (vladdraql) [inline capuccino/capuccino_py.py]: kewl

File: espresso/espresso_py.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from math import pi, floor
+
+class espresso:
+    cantitate_totala_cafea = 0
```
Comment (mihaipotoroaca) [inline espresso/espresso_py.py]: english needed

File: espresso/espresso_py.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        espresso.time = espresso.cantitate_totala_cafea / pi
+        return floor(espresso.time)
+
+    @staticmethod
+    def afisare():
+        print(f'Cantitatea totala de espresso: {espresso.cantitate_totala_cafea} grame')
+        print(f'Timpul de preparare espresso: {floor(espresso.time)} secunde')
+
+if __name__ == '__main__':
+    e = espresso(80, 10, 50)
```
Comment (mihaipotoroaca) [inline espresso/espresso_py.py]: use more eloquent naming

**Review by mihaipotoroaca — COMMENTED** [review]
Seems ok

File: capuccino/capuccino_py.py (hunk 1 above)
Comment (CosminTraistaru) [inline capuccino/capuccino_py.py]: Class names should normally use the CapWords convention.
https://www.python.org/dev/peps/pep-0008/#class-names

File: capuccino/capuccino_py.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    def __init__(self, c_coffee, c_sugar, c_water,  milk):
+        super().__init__(coffee = c_coffee, sugar = c_sugar, water = c_water)
```
Comment (CosminTraistaru) [inline capuccino/capuccino_py.py]: When calling methods you should not use spaces before and after = 
https://www.python.org/dev/peps/pep-0008/#other-recommendations

File: capuccino/capuccino_py.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
+from math import *
```
Comment (CosminTraistaru) [inline capuccino/capuccino_py.py]: Wildcard imports (from <module> import *) should be avoided
https://www.python.org/dev/peps/pep-0008/#imports

File: capuccino/capuccino_py.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+from espresso.espresso_py import espresso
+
+class capuccino(espresso):
+    def __init__(self, c_coffee, c_sugar, c_water,  milk):
+        super().__init__(coffee = c_coffee, sugar = c_sugar, water = c_water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        super().prepare_coffee()
+        self.add_milk = espresso.cantitate_totala_cafea + self.milk
```
Comment (CosminTraistaru) [inline capuccino/capuccino_py.py]: If you use this variable only in this method you don't have to assign it to the object (you don't have to use self)

File: espresso/espresso_py.py (hunk 2 above)
Comment (CosminTraistaru) [inline espresso/espresso_py.py]: What happens if you have multiple objects instantiated from this class?  
Will the `cantitate_totala_cafea` variable save the correct value for each object?

File: espresso/espresso_py.py (hunk 3 above)
Comment (flavia-c) [reply espresso/espresso_py.py]: it's ok within the actual context

File: espresso/espresso_py.py (hunk 2 above)
Comment (flavia-c) [reply espresso/espresso_py.py]: amended

File: capuccino/capuccino_py.py (hunk 1 above)
Comment (flavia-c) [reply capuccino/capuccino_py.py]: amended

File: capuccino/capuccino_py.py (hunk 4 above)
Comment (flavia-c) [reply capuccino/capuccino_py.py]: amended

File: capuccino/capuccino_py.py (hunk 5 above)
Comment (flavia-c) [reply capuccino/capuccino_py.py]: amended

File: capuccino/capuccino_py.py (hunk 6 above)
Comment (flavia-c) [reply capuccino/capuccino_py.py]: amended

File: espresso/espresso_py.py (hunk 2 above)
Comment (flavia-c) [reply espresso/espresso_py.py]: added the variable as a reference to the object (self, not global anymore).

File: capuccino/capuccino_py.py [hunk 7]
```diff
@@ -0,0 +1,28 @@
+from math import sqrt, floor, pow
```
Comment (cameliaenache) [inline capuccino/capuccino_py.py]: If you have to enumerate a lot of functions, it is easier to use: import math

File: capuccino/capuccino_py.py [hunk 8]
```diff
@@ -0,0 +1,27 @@
+import math
```
Comment (CosminTraistaru) [inline capuccino/capuccino_py.py]: why is this better than:
`from math import sqrt, floor, pow` ??

File: capuccino/capuccino_py.py (hunk 7 above)
Comment (flavia-c) [reply capuccino/capuccino_py.py]: Amended, so that it would be clear from which library the functions are coming from.

**Review by CosminTraistaru — APPROVED** [review]

**Review by vladdraql — APPROVED** [review]

**Review by mihaipotoroaca — APPROVED** [review]

**Review by miutesculazar — APPROVED** [review]

**Review by cameliaenache — APPROVED** [review]

**Pull request merged**